plugin/trino-vertica/src/test/java/io/trino/plugin/vertica/TestingVerticaServer.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.vertica;
+
+import com.google.common.collect.ImmutableSet;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import java.util.Set;
+
+import static java.lang.String.format;
+
+public class TestingVerticaServer
+        extends JdbcDatabaseContainer<TestingVerticaServer>
+{
+    public static final Integer PORT = 5433;
+    private static final String DATABASE = "test_db";
+    public static final String SCHEMA = "tpch";
+    private static final String USER = "test_user";
+    private static final String PASSWORD = "test_password";
+
+    public TestingVerticaServer()
+    {
+        this("datagrip/vertica:8.1.1");
```

Comment:
Why not 9?

Reply:
Somehow I was missing the new version. Changed to 9.1.1.